src/ImageSharp/Formats/Tga/TgaImageType.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.
+    ImageSharp.Formats.Tga
+{
+    /// <summary>
+    /// Defines the tga image type. The TGA File Format can be used to store Pseudo-Color,
+    /// True-Color and Direct-Color images of various pixel depths.
+    /// </summary>
+    public enum TgaImageType : byte
+    {
+        /// <summary>
+        /// No image data included.
+        /// Not sure what this is used for.
```

Comment:
Let's remove this comment from the summary. I don't think the type is ever used but it's in the spec so no need for the comment.